closed/src/jdk.crypto.ec/share/classes/sun/security/ec/NativeECUtil.java

```diff
@@ -107,7 +107,7 @@ static String getCurveName(ECParameterSpec params) {
         String curveName;
         if (params instanceof NamedCurve) {
             NamedCurve namedCurve = (NamedCurve) params;
-            curveName = namedCurve.getName();
+            curveName = namedCurve.toString();
```

Comment:
This doesn't yield the name of the curve; this should be
```java
            curveName = namedCurve.getNameAndAliases()[0];
```
(see the [head stream](https://github.com/ibmruntimes/openj9-openjdk-jdk/blob/openj9/closed/src/java.base/share/classes/sun/security/ec/NativeECUtil.java#L109)).

Reply:
Fixed